package.json

```diff
@@ -107,5 +107,6 @@
     "cross-spawn": "^7.0.6",
     "micromatch": "4.0.6",
     "rollup": "^4.24.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Typically enforcing packageManager is not necessary for our services, I'm guessing this is accidental 🤔